code/src/Components/MessageList.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react'
+
+import moment from 'moment'
+
+import { LikedMessage } from 'Components/LikedMessage'
+import './MessageList.css'
+
+export const MessageList = () => {
+  const MessagesUrl = 'https://happy-thoughts-technigo.herokuapp.com/thoughts'
+  const [messages, setMessages] = useState([])
+
+  useEffect(() => {
+    fetch(MessagesUrl)
+      .then((res) => {
+        return res.json()
+      })
+      .then((data) => {
+        console.log(data)
+        // filter out empty messages 
+        const filteredMessages = data.filter((message) => message.message)
+        // save data to state
+        setMessages(filteredMessages)
+      })
+  }, [messages]) // this updates the content of the page without reloading it!
```

Comment:
I don't think your useEffect is working correctly as there is an infinite loop coming from your console log on line 18. I think removing "messages" and having an empty array would fix this.

Reply:
Exactly what I wanted to say, great catch Jamie. Needs to be fixed

Reply:
Fixed